rook-ceph-aio/values.yaml

```diff
@@ -1,248 +1,180 @@
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r008c013
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r008c014
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r008c015
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r008c016
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r008c017
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r008c018
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r008c019
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r008c020
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c001
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c002
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c003
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c004
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c005
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c006
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c007
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c008
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c009
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c010
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c011
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c012
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c013
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c014
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c015
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c016
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c017
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c018
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c019
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
-        # - name: rdm9r009c020
-        #   devices:
-        #   - name: sdd
-        #   - name: sde
+            - name: node1
```

Comment:
we probably want to leave these commented out

Reply:
complete